src/pages/instances/InstanceOverviewDeviceList.tsx

```diff
@@ -0,0 +1,63 @@
+import { FC } from "react";
+import { LxdInstance } from "types/instance";
+import { isRootDisk } from "util/instanceValidation";
+import { FormDevice } from "util/formDevices";
+import { LxdDeviceValue, LxdGPUDevice } from "types/device";
+import ResourceLink from "components/ResourceLink";
+
+interface Props {
+  device: LxdDeviceValue;
+  instance: LxdInstance;
+}
+
+const InstanceOverviewDeviceList: FC<Props> = ({ device, instance }) => {
+  return (
+    <div className="u-truncate">
```

Comment:
This `div` is not necessary, you can add `className`  attributes to the column object properties i.e. where this component is being called.